test/NuGet.Core.FuncTests/NuGet.XPlat.FuncTest/Util.cs

```diff
@@ -0,0 +1,151 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using NuGet.Test.Utility;
+using Xunit;
+
+namespace NuGet.XPlat.FuncTest
+{
+    internal class Util
+    {
+        private static string DotnetCliBinary { get; set; }
+
+        private static string DotnetCliExe { get; set; }
+
+        private static string XplatDll { get; set; }
+
+        private static string XplatDllShell { get; set; }
+
+        static Util()
+        {
+            DotnetCliBinary = @"dotnet";
+            DotnetCliExe = @"dotnet.exe";
+            XplatDll = @"NuGet.Core\NuGet.CommandLine.XPlat\bin\release\netcoreapp1.0\NuGet.CommandLine.XPlat.dll";
+            XplatDllShell = @"NuGet.Core/NuGet.CommandLine.XPlat/bin/release/netcoreapp1.0/NuGet.CommandLine.XPlat.dll";
```

Comment:
Same path just with different slashes? Use `static readonly string` with `Path.Combine` so you only need one field.